maintenance-filter-test-webapp/pom.xml

```diff
@@ -39,6 +39,12 @@
       <artifactId>log4j</artifactId>
       <version>1.2.14</version>
     </dependency>
+
+    <dependency>

```

Comment:
How did tests previously run without this (i.e. in https://github.com/CruGlobal/maintenance-filter/commit/008a6073a92095899af014874e2f0e2144037530)?

Reply:
it's a manual test. I must not have run the test manually at that time.